app/javascript/data/advancedFilterOptions.js

```diff
@@ -454,4 +412,70 @@ export const advancedFilterOptions = [
       },
     ],
   },
+  {
+    name: 'address',
+    title: 'Address (Combination)',
+    description: 'Monitorees with specified address',
+    type: 'combination',
+    fields: [
+      {
+        name: 'address-foreign',
+        title: 'outside USA',
+        description: 'Monitoree Address 1, Town/City, Country, Address 2, Postal Code, Address 3 or State/Province (outside USA)',
+        type: 'search',
+      },
+      {
+        name: 'address-usa',
+        title: 'within USA',
+        description: 'Monitoree Address 1, Town/City, State, Address 2, Zip, or County within USA',
+        type: 'search',
+      },
+    ],
+  },
+  {
+    name: 'name',
+    title: 'Name (Combination)',
+    description: 'Monitoree name',
+    type: 'combination',
+    fields: [
+      {
+        name: 'first-name',
+        title: 'first',
+        description: 'Monitoree first name',
+        type: 'search',
+      },
+      {
+        name: 'last-name',
+        title: 'last',
+        description: 'Monitoree last name',
+        type: 'search',
+      },
+      {
+        name: 'middle-name',
+        title: 'middle',
+        description: 'Monitoree middle name',
+        type: 'search',
+      },
+    ],
+  },
+  {
+    name: 'telephone',
```

Comment:
The telephone one is supposed to stay type search and utilize the `additionalFilterOption` - right @hackrm?

Reply:
@hackrm is on PTO but this is what the ticket says:
```
Telephone (exact match/contains):
Combine these filters to be the same format as the current "Close Contact with a Known Case ID" filter option that has a drop down for the exact match/contains options.
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1179/commits/a3e0e0e903d1aca09d9c4eb75e0bd704b0ab3a18